x-pack/packages/kbn-inference-endpoint-ui-common/kibana.jsonc

```diff
@@ -0,0 +1,12 @@
+{
+  "type": "shared-browser",
+  "id": "@kbn/inference-endpoint-ui-common",
+  "owner": [
+    "@elastic/response-ops",
+    "@elastic/appex-ai-infra",
+    "@elastic/obs-ai-assistant",
+    "@elastic/security-generative-ai"
+  ],
+  "group": "platform",
+  "visibility": "shared"
+}
```

Comment:
If this `package` is going to be used from both `Search` solution and connectors, then I believe it makes sense to categorise it as _platform/shared_ like you did.

However, the package should be placed under `x-pack/platform/packages/shared`.
Can you relocate it and update the PR?

Reply:
@gsoldevila : moved the package into `x-pack/platform/packages/shared`